projects/canopy/src/lib/forms/sort-code/sort-code.stories.ts

```diff
@@ -5,25 +5,25 @@ import { moduleMetadata } from '@storybook/angular';
 import { boolean, text, withKnobs } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 
-import { LgSortCodeComponent } from './sort-code.component';
 import { CanopyModule } from '../../canopy.module';
 import { notes } from './sort-code.notes';
+import { LgSortCodeDirective } from './sort-code.directive';
 
 @Component({
   selector: 'lg-reactive-form',
   template: `
     <form [formGroup]="form">
-      <lg-sort-code formControlName="sortCode" [focus]="focus">
+      <lg-input-field>
         {{ label }}
         <lg-hint *ngIf="hint">{{ hint }}</lg-hint>
-      </lg-sort-code>
+        <input lgInput lgSortCode formControlName="sortCode" />
```

Comment:
~~Do we need to keep the `focus` input?~~ Ah I see it was never actually used

Reply:
Yes, and also the `focus` was an input to the `lg-sort-code` component but it's not needed by the directive because we can simply add the `lgFocus` directive straight on the input :)